web-registry/src/types/ledger/ecocredit.ts

```diff
@@ -73,6 +73,73 @@ export interface QueryBasketsResponse {
   pagination?: PageResponse;
 }
 
+export interface IBasket {
+  id: string;
+  basketDenom: string;
+  name: string;
+  disableAutoRetire: boolean;
+  creditTypeAbbrev: string;
+  dateCriteria: {
+    minStartDate?: string;
+    startDateWindow?: string;
+  };
+  exponent: string;
+}
+
+export interface QueryBasketRequest {
+  basketDenom: string;
+}
+
+export interface QueryBasketResponse {
+  basket?: IBasket;
+  classes: string[];
+}
+
+export interface QueryBasketBalancesRequest {
+  basketDenom: string;
+  // pagination?: PageRequest; // DISABLED
```

Comment:
why disabling them? 
it was already optional

Reply:
Precisely, since it is optional and we do not use it, and these interface implementations are provisional, I thought that it did not make much sense to import or implement the PageRequest interface (which is currently not there, the one we have available is PageResponse). What should I do?

Reply:
ok fine for now now then

Reply:
As I am finally going to push some changes, it costs me nothing to implement that PageResponse and thus we leave the interfaces of those 2 queries that optionally incorporate it complete. okay?

Reply:
sounds good